bin/install-wp-tests.sh

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env bash
```

Comment:
The Github action can not execute this file : https://github.com/studiometa/wp-toolkit/runs/1551983719?check_suite_focus=true#step:7:7

@lusimeon did you set its chmod correctly? 🤔